srcpkgs/mpDris2/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'mpDris2'
+pkgname=mpDris2
+version=0.8
+revision=1
+build_style=gnu-configure
+hostmakedepends="autoconf automake intltool gettext-devel"
+depends="python3 python3-dbus python3-gobject python3-mpd2 python3-mutagen"
+short_desc="Provides MPRIS 2 support to mpd (Music Player Daemon)"
+maintainer="Illia Shestakov <ishestakov@airmail.cc>"
+license="GPL-3.0-only"
+homepage="https://github.com/eonpatapon/mpDris2"
+distfiles="https://github.com/eonpatapon/mpDris2/archive/${version}.tar.gz"
+checksum=b6b15c1fdddf16a6d74485ad09f56ed353a317e149c37475c00a279186da4391
+
+pre_configure() {
+	./autogen.sh --sysconfdir=/etc --prefix=/usr
```

Comment:
Can autogen.sh be executed with `NOCONFIGURE=1` for this package? If that is the case, it would be better to use that and leave the configure step to the build_style function.

Reply:
Looks like it works.